Week-13/weather/routes/routes.js

```diff
@@ -0,0 +1,62 @@
+const express = require("express");
+//const Model = require("../models/model");
+const router = express.Router();
+const axios = require("axios");
+require("dotenv").config();
+const API_KEY = process.env.API_KEY;
+
+router.get("/current", async (req, res) => {
+
+  const result = [];
+  const cities = new Set(req.query.city.split(","));
+  console.log(cities);
+  for (let city of cities) {
+    console.log("City" + city)
+    // let url = `http://api.weatherapi.com/v1/current.json?key=${API_KEY}&q=${city}`;
+    let url = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${process.env.API_KEY}`
+    try {
+      let results = await axios.get(url);
```

Comment:
As discussed this should be implemented using promis.all

Reply:
Changes done